101-vm-simple-rhel/azuredeploy.json

```diff
@@ -24,25 +24,31 @@
       "metadata": {
         "description": "The RHEL version for the VM. This will pick the latest available image of this given RHEL version."
       }
+    },
+    "vmName": {
+      "type": "string",
```

Comment:
vmName is necessary because we want the user to be able to create multiple VMs with the same template. Otherwise subsequent invocations will fail.